src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile2.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+
+// Given_StorageFile2 , as Given_StorageFile is used in PR#2407
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+    [TestClass]
+    public class Given_StorageFile2
+    {
+        String _filename;
+        
+        [TestInitialize]
+        public void Init()
+        {
+            _filename = DateTime.Now.ToString("yyyyMMddHHmmss") + ".txt";
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+        }
+
+        [TestMethod]
+        public async void When_DateCreated()
+        {
+            var folderForTestFile = Windows.Storage.ApplicationData.Current.LocalFolder;
+            Assert.IsNotNull(folderForTestFile, "cannot get LocalFolder - error outside tested method");
+
+            Windows.Storage.StorageFile testFile = null;
+            
+            DateTimeOffset dateBeforeCreating = DateTimeOffset.Now;
+      
+            try
+            {
+                testFile = await folderForTestFile.CreateFileAsync( _filename, Windows.Storage.CreationCollisionOption.FailIfExists);
+                Assert.IsNotNull(testFile, "cannot create file - error outside tested method");
+            }
+            catch
+            {
+                  Assert.Fail("CreateFile exception - error outside tested method");
+            }
+
+            DateTimeOffset dateAfterCreating = DateTimeOffset.Now;
+
+            // test of DateCreated
+
+            // first, some wait - to be sure that returned date is not simply 'current date'
+            // e.g. FAT has two seconds resolution - so wait should be longer
+            await Task.Delay(5000);
```

Comment:
Most of the OSs don't have that kind of resolution. Don't wait arbitrarily, and if you do need to wait, at least make a small loop until it really becomes different. Adding 5s for every test that validates this behavior is too expensive for the CI.

Reply:
without this wait, test would pass even when DateCreated is current date. Test have to wait a bit more than twice than file system date resolution.
I can remove this test, leaving only test for DateCreate not being default date (like year 1601, or 1970).
Remove this test?

Reply:
You should be able to test this with less delay, or make a loop with a shorter delay to validate that the date changed.

Reply:
Do you mean loop like `{delay(100); if(date.created < datetime.current) then OK}`?
but date.created can be < datetime.current even on first run, e.g. when datetime.current.secs=31, and date.created is with 2 sec resolution - date.created would be = 30. Or, maybe 32, it depends on rounding :)
Also, in external storages world, date.created can be set with respect to storage time, not system time - and these times can be something like out of sync. And all tests would be bad :)

Reply:
In this context, the validation is done on local storage, using a stable emulator; there should not be much variations. A loop as you described would be enough.